### PR TITLE
Add reclaim policy to kubectl get storage class result

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

 /kind feature

**What this PR does / why we need it**:

Currently, kubectl get sc shows too few features. IMO, reclaim policy is important for users.
I would like add reclaim policy to kubectl get storage class result

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
ping @soltysh

File: staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go [hunk 1]
```diff
@@ -1453,6 +1453,7 @@ func (t *Tester) testListTableConversion(obj runtime.Object, assignFn AssignFunc
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
 	}
+	t.Logf("%+v", table)
```
Comment (soltysh) [inline staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go]: Please remove this debug statement.

File: pkg/printers/internalversion/printers.go [hunk 2]
```diff
@@ -410,6 +410,7 @@ func AddHandlers(h printers.PrintHandler) {
 	storageClassColumnDefinitions := []metav1beta1.TableColumnDefinition{
 		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
 		{Name: "Provisioner", Type: "string", Description: storagev1.StorageClass{}.SwaggerDoc()["provisioner"]},
+		{Name: "ReclaimPolicy", Type: "string", Description: storagev1.StorageClass{}.SwaggerDoc()["reclaimPolicy"]},
```
Comment (soltysh) [inline pkg/printers/internalversion/printers.go]: I spoke with storage and they suggested adding also `AllowVolumeExpansion` and `VolumeBindingMode` wdyt?

**Review by soltysh — CHANGES_REQUESTED** [review]

File: pkg/printers/internalversion/printers.go (hunk 2 above)
Comment (cwdsuzhou) [reply pkg/printers/internalversion/printers.go]: Thanks for view, I would  add it. 

I also think it is needed, `get result` of StorageClass is too simple  now, it just contains the name and provisioner.

Comment (cwdsuzhou) [reply pkg/printers/internalversion/printers.go]: @soltysh I have updated the PR

File: pkg/printers/internalversion/printers.go [hunk 3]
```diff
@@ -1857,7 +1860,23 @@ func printStorageClass(obj *storage.StorageClass, options printers.GenerateOptio
...
+		volumeBindingMode = string(*obj.VolumeBindingMode)
+	}
+
+	allowVolumeExpansion := false
+	if obj.AllowVolumeExpansion != nil {
+		allowVolumeExpansion = *obj.AllowVolumeExpansion
+	}
+
+	row.Cells = append(row.Cells, name, provtype, reclaimPolicy, volumeBindingMode, allowVolumeExpansion,
+		translateTimestampSince(obj.CreationTimestamp))
```
Comment (cwdsuzhou) [inline pkg/printers/internalversion/printers.go]: According to our docs, 
default value of ReclaimPolicy is `Delete`
VolumeBindingImmediate is `Immediate`
VolumeBindingImmediate is `false`

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**